examples/lighting-app/nrfconnect/CMakeLists.txt

```diff
@@ -20,7 +20,13 @@ get_filename_component(CHIP_APP_SERVER ${CHIP_ROOT}/examples/common/chip-app-ser
 get_filename_component(NRFCONNECT_COMMON ${CHIP_ROOT}/examples/platform/nrfconnect REALPATH)
 get_filename_component(LIGHTING_COMMON ${CHIP_ROOT}/examples/lighting-app/lighting-common REALPATH)
 
-set(CONF_FILE ${CHIP_ROOT}/config/nrfconnect/sample-defaults.conf prj.conf)
+# RPC supporting version can be built with this command:
+#
+# west build -b nrf52840dk_nrf52840 -- -DCONF_FILE='prj_with_rpc.conf;../../../config/nrfconnect/sample-defaults.conf'
```

Comment:
zephyr also supports config overlays which are applied on top of the main configuration:
west build -b nrf52840dk_nrf52840 -- -DOVERLAY_CONFIG=rpc.conf

Reply:
Nice, changed the PR to provide rpc.overlay instead of full .conf